ads/google/a4a/shared/shared-util.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @fileoverview Split of content-recommendation.js
```

Comment:
I don't think you need to comment that this split off of ads/google/a4a/shared/content-recommendation.js